exercises/circular-buffer/src/lib.rs

```diff
@@ -1 +1,44 @@
+use std::fmt::Debug;
+use std::marker::PhantomData;
 
+pub struct CircularBuffer<T: Debug> {
+    // This field is here to make the template compile and not to
+    // complain about unused type parameter 'T'. Once you start
+    // solving the exercise, delete this field and the 'std::marker::PhantomData'
+    // import.
+    field: PhantomData<T>,
+}
+
+#[derive(Debug, PartialEq)]
+pub enum Error {
+    EmptyBuffer,
+    FullBuffer,
+}
+
+impl<T: Debug> CircularBuffer<T> {
```

Comment:
same comment as with `sublist` - I know that this `Debug` constraint is added purely so that we can reference variables in `unimplemented` message. We made a different choice in https://github.com/exercism/rust/commit/c5f7a433c5e954aa78162bfcab14eab96510bc4f and I'll consider whether it's better to use underscore prefix here. Input appreciated.

Reply:
I think that we should not bound `T: Debug` here, and instead use an underscore variable. Perhaps it's worth identifying which of the exercises first uses an underscore variable in the track and adding a comment explaining why that is used, instead of a `Debug` trait bound.